src/show/components/ConstructorCard/index.jsx

```diff
@@ -12,4 +14,12 @@ const ConstructorCard = ({ constructor }) => {
   );
 };
 
+ConstructorCard.defaultProps = {
+  constructor: undefined
```

Comment:
Why is this undefined, just define it as an empty object.

Reply:
done